crocket/src/main/java/com/crocket/Player.java

```diff
@@ -0,0 +1,71 @@
+package com.crocket;
+
+import java.util.Queue;
+import java.util.LinkedList;
+
+
+public class Player implements IEventListener{
+    int strokes;
+    boolean finished;
+    Queue<Entity> unpassedHoops; //Queue of hoops that have not been passed through yet
+    Ball ball;
+    
+    Player(Ball ball) {
+        this.unpassedHoops =  new LinkedList<Entity>(); 
+        this.strokes = 0;
+        this.finished = false;
+        this.ball = ball;
+
+    }
+
+    //Checks if the player has passed through all hoops
+    public boolean hasWon() {
```

Comment:
For a player to win, they need to hit a peg as well, or am I mistaken?

Reply:
Will be fixed if unpassedHoops gets changed

Reply:
unpassedHoops has been changed to remainingTarget